composer.json

```diff
@@ -12,15 +12,16 @@
             "homepage": "http://php-and-symfony.matthiasnoback.nl"
         }
     ],
+    "minimum-stability": "beta",
```

Comment:
The `prefer-stable` option should be added in this case.

Reply:
done.

Reply:
I would not add this setting here, but use `composer config minimum-stability beta` in the Travis CI config instead. Setting `prefer-stable` to `true` would have the effect of not ending up with Symfony 3.4 or 4.0 right now (but you would get 3.3.13 instead).